modules/rpc/internal/stats/metrics.go

```diff
@@ -33,27 +33,47 @@ const (
 	TagMiddleware = "middleware"
 )
 
-// HTTPTags creates metrics scope with defined tags
 var (
-
-	// RPCTags creates metrics scope with defined tags
-	RPCTags = map[string]string{
+	rpcTags = map[string]string{
 		TagModule: "rpc",
 		TagType:   "request",
 	}
+)
 
+// Client is a client for stats.
+type Client interface {
```

Comment:
I really like putting things behind interfaces in general. In the future, `stats.Client` might be something more complicated than what amounts to a struct of counters, and I always default to using interfaces over structs for actual logic so that I can swap them out in testing.